backend/authentication/models/user.py

```diff
@@ -192,3 +194,55 @@ def get_associations_with_permission(self, permission: str) -> set:
         from associations.models.role import Role
 
         return set(Role.objects.filter(query).values_list("association__pk", flat=True))
+
+    def get_absolute_url(self):
+        return '/profils/'+self.id
+
+    ## For students' genealogy ##
+
+    def genealogy_relations(self):
+        """All neighbors of a student in the Minor's graph"""
+        successors = []
+        if self.roommate.all:
```

Comment:
I believe `all` is a function here, so if that is the case, I'm not sure what we are testing here